app/controllers/v0/coronavirus_chatbot/tokens_controller.rb

```diff
@@ -52,7 +52,7 @@ def token_endpoint_uri
         return @token_uri if @token_uri.present?
 
         token_endpoint = "https://#{Settings.coronavirus_chatbot.directline_uri || 'directline.botframework.com'}" \
-          '/v3/directline/tokens/generate'
+                         '/v3/directline/tokens/generate'
```

Comment:
what's with all the extra whitespace?

Reply:
New rubocop version added a check that string literals align.